### PR TITLE
Test for PR #194

### PR DESCRIPTION
**Pull request by gam6itko**
Test for PullRequest #194. 
No fail if `$countQuery->getQuery()->getSingleScalarResult()` returns string

Changelog
### Added
- Added some `PagerTest::testComputeNbResult` to test int return

### CODE REVIEW

File: tests/Pager/Doctrine/PagerTest.php [hunk 1]
```diff
@@ -63,4 +67,40 @@ public function testGetResults(): void
 
         $this->assertSame($expectedObjects, $this->pager->getResults());
     }
+
+    /**
+     * Test ensure getSingleScalarResult result within computeNbResult will be cast.
+     *
+     * @covers ::computeNbResult
```
Comment (OskarStark) [inline tests/Pager/Doctrine/PagerTest.php]: this can be removed, as we don't us e`@covers` annotations

File: tests/Pager/Doctrine/PagerTest.php [hunk 2]
```diff
@@ -13,13 +13,17 @@
...
 use Doctrine\ORM\Query;
+use Doctrine\ORM\QueryBuilder;
 use PHPUnit\Framework\TestCase;
 use Sonata\DatagridBundle\Pager\Doctrine\Pager;
 use Sonata\DatagridBundle\ProxyQuery\ProxyQueryInterface;
 
 /**
  * @author Romain Mouillard <romain.mouillard@gmail.com>
+ *
+ * @coversDefaultClass \Sonata\DatagridBundle\Pager\Doctrine\Pager
```
Comment (OskarStark) [inline tests/Pager/Doctrine/PagerTest.php]: this can be removed, as we don't us e`@covers` annotations

Comment (gam6itko) [reply tests/Pager/Doctrine/PagerTest.php]: done

**Comment by OskarStark** [comment]
Thank you

**Pull request merged**